src_assets/common/assets/web/troubleshooting.html

```diff
@@ -39,16 +35,26 @@ <h2>Unpair All Clients</h2>
         Error while unpairing
       </div>
       <div>
-        <button
-          class="btn btn-danger"
-          :disabled="unpairAllPressed"
-          @click="unpairAll"
-        >
+        <button class="btn btn-danger" :disabled="unpairAllPressed" @click="unpairAll">
           Unpair All
         </button>
       </div>
     </div>
   </div>
+  <!--Logs-->
+  <div class="card p-2 my-4">
+    <div class="card-body">
+      <h2>Logs</h2>
+      <br />
+      <div class="d-flex justify-content-between align-items-baseline py-2">
+        <p>See the logs uploaded by Sunshine</p>
+        <input type="text" class="form-control" v-model="logFilter" placeholder="Find..." style="width: 300px">
```

Comment:
I like the idea of this feature! Is it difficult to add a copy button, like this?
![image](https://user-images.githubusercontent.com/42013603/210084899-d099be41-88e4-4785-9396-5d708f2cde9e.png)

Wouldn't necessarily need to be inside the log text container.

Reply:
Done